nginx_ingress/lego/rbac.yaml

```diff
@@ -1,32 +1,49 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: kube-lego-serviceaccount
```

Comment:
You changed the service account name, but there are still resources that reference the old name, such as the lego deployment.